purchases/src/main/kotlin/com/revenuecat/purchases/common/events/EventsManager.kt

```diff
@@ -103,6 +104,7 @@ internal class EventsManager(
 
             val backendEvent = when (event) {
                 is PaywallEvent -> event.toBackendStoredEvent(identityManager.currentAppUserID)
+                is CustomerCenterEvent -> event.toBackendStoredEvent(identityManager.currentAppUserID)
```

Comment:
Hmm I'm wondering if the `toBackendStoredEvent` could be moved to the `FeatureEvent` interface, so we're forced to implement them and we don't need need to do this separation.

Reply:
Love it

Reply:
Thought about that while working on it. I'll apply it in this PR then.

Reply:
One caveat is that we would need to make `BackendStoredEvent` public, which I am not sure we want to. Unless I can make something like a private protocol extension in swift? 🤔

Reply:
Ohh right, I didn't think about that... I don't think you can add an extension to an interface like that :( 
I guess we could instead create a sealed interface with a private extension and a computed property to do this... But we are trying to avoid those sealed types since they can cause breaking changes if new types are added... 

So I actually think your current solution might be better then, to avoid exposing more API without need... Please ignore my comment 🙇